src/voting.py

```diff
@@ -3,24 +3,24 @@
 
 from nextcord import Message
 
-import bust
 import config
 import song_utils
+from bust import BustController
 
 # TODO: Eventually the function in this file should be rewritten not interact with
 # Discord at all, and instead called by a wrapper in bust.py with most
 # internal logic remaining here
 
 
-async def command_form(
-    message: Message, google_drive_image_link: Optional[str] = None
+async def form(
```

Comment:
I might give these methods slightly more informative names while we're here. Such as:

* `generate_form`
* `skip_song`
* `send_song_list`

Since they're no longer directly tied to the command names.